### PR TITLE
Math.whatever -> precision

### PR DESCRIPTION
**Pull request by samgermain**
I'm trying to finally move everything over to fixed point math

I've had some trouble creating accounts on these exchanges. If any of @ttodua @sc0Vu @pcriadoperez @carlosmiei @DoctorSlimm have api keys for any of these exchanges and could test that exchange that would be appreciated

- [ ] cdax #14857
- [ ] cex #14858
- [x] exmo #14859
- [ ] huobijp #14875
- [ ] luno #14860
- [ ] ripio #14861
- [ ] stex #14862
- [ ] upbit #14873
- [ ] xena #14871

### CODE REVIEW

File: js/exmo.js [hunk 1]
```diff
@@ -1758,22 +1758,22 @@ module.exports = class exmo extends Exchange {
         // fixed funding fees only (for now)
         if (!this.fees['transaction']['percentage']) {
             const key = (type === 'withdrawal') ? 'withdraw' : 'deposit';
-            let feeCost = this.safeNumber (transaction, 'commission');
+            let feeCost = this.safeString (transaction, 'commission');
             if (feeCost === undefined) {
-                feeCost = this.safeNumber (this.options['transactionFees'][key], code);
```
Comment (pcriadoperez) [inline js/exmo.js:1763]: `fetchDeposits` works, however `fetchTransactions` throws an error when there is no transaction fee for this code.

This error is unrelated to this change, and the change you propose here I think is correct. I'll open a different PR to fix it.

Comment (samgermain) [reply js/exmo.js:1763]: thanks

**Comment by frosty00** [comment]
i suggest to test these changes by setting `exchange.number = decimal.Decimal` and checking that the method in question still works

File: js/exmo.js (hunk 1 above)
Comment (samgermain) [reply js/exmo.js:1763]: @pcriadoperez Could you actually separate stex and exmo into separate PRs?

**Comment by samgermain** [comment]
Reopened in separate PRs for each exchange

**Pull request closed without merging**